app/Platform/Controllers/SuggestGameController.php

```diff
@@ -293,7 +293,7 @@ private function getReasonColumn(): array
         ];
     }
 
-    public function isGameEligible(int $gameId, array $gameIds): bool
+    public function isGameEligible(int $gameId, array $gameIds, array $why = []): bool
```

Comment:
i think `$context` would be a more fitting name for this parameter. not that important though